CHANGELOG.rst

```diff
@@ -1,3 +1,12 @@
+Ndcube v2.0.0
```

Comment:
This change should be removed.  The entry in this file is automatically built my `towncrier` from the `323.feature.rst` file at the time of release.

Reply:
Yup, sure

Reply:
Done